deltacat/compute/compactor_v2/constants.py

```diff
@@ -42,7 +42,7 @@
 PARQUET_TO_PYARROW_INFLATION = 4
 
 # By default, copy by reference is enabled
```

Comment:
Comment would need update.

Not a comment on this CR, and it's likely infeasible to change at this point -- but it's better to have flags and variables that enable rather than negate; i.e. `ENABLE_COPY_BY_REFERENCE` rather than `DISABLE_COPY_BY_REFERENCE`. Else you get confusing double-negatives like `DISABLE_COPY_BY_REFERENCE = False`.